File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/Switch_Case/menubased.cpp
//Menu Based program depicting how switch case works 

#include<iostream>
using namespace std;

int main()
{
        int n;

        cout<<"      MENU     "<<endl;
        cout<<"1. Tea"<<endl;
        cout<<"2. Coffee"<<endl;
        cout<<"3. Juice"<<endl;
        cout<<"4. Exit"<<endl;

        cout<<"Enter your choice: "<<endl;
        cin>>n;

        switch (n) 
        {
            case 1:
                cout<<"You selected Tea";
                break;

            case 2:
                cout<<"You selected Coffee";
                break;

            case 3:
                cout<<"You selected Juice";
                break;

            case 4:
                cout<<"Thank you! Exiting...";
                break;

            default:
                cout<<"Invalid choice";
        }

    }
    

File: 00_Learn_The_Basics/Things_To_Know_In_A_Language/Switch_Case/MenuBased.java
//Menu Based program depicting how switch case works 

import java.util.*;

public class MenuBased
{
    public static void main(String[] args)
    {
        Scanner in=new Scanner(System.in);
        
        System.out.println("      MENU     ");
        System.out.println("1. Tea");
        System.out.println("2. Coffee");
        System.out.println("3. Juice");
        System.out.println("4. Exit");

        System.out.print("Enter your choice: ");
        int n=in.nextInt();

        switch (n) 
        {
            case 1:
                System.out.println("You selected Tea");
                break;

            case 2:
                System.out.println("You selected Coffee");
                break;

            case 3:
                System.out.println("You selected Juice");
                break;

            case 4:
                System.out.println("Thank you! Exiting...");
                break;

            default:
                System.out.println("Invalid choice");
        }

        in.close();
    }
}
    
